ferris_cast_core/src/lib.rs

```diff
@@ -1,14 +1,76 @@
-pub fn add(left: usize, right: usize) -> usize {
-    left + right
+use anyhow::{anyhow, Result};
+use chrono::prelude::DateTime;
+use chrono::{NaiveDate, ParseError, Utc};
+use rss::{Channel, Item};
+use std::convert::From;
+use std::fs::File;
+use std::io::BufReader;
+use std::path::PathBuf;
+use std::vec::Vec;
+
+pub static BAD_STRING: &str = "Unknown";
+
+pub struct Podcast {
+    pub title: String,
+    pub link: String,
+    pub image: rss::Image,      // TODO: If necessary add an image if needed
```

Comment:
Make this an Option
```suggestion
    pub image: Option<rss::Image>,      // TODO: If necessary add an image if needed
```

Reply:
I just made this into a String as we don't really need the other fields of Image.